models/risk_models.py

```diff
@@ -6,18 +6,24 @@
 from utils import _encode_message, _decode_message, binary_to_float
```

Comment:
Not sure if those lines are needed but they will not work in a unittest runner which has it's own working dir
```
import sys
import os
sys.path.append(os.getcwd())
```

Reply:
it's needed to run from root like `python models/run.py --data_path`.

We need to ship this as a python package with setup.py, which is a much better solution, but not within the scope of this PR.

Reply:
I'm not sure how to fix this except to package the repo...